isomerpages/isomercms-backend
# build(app): adds typescript to backend

## Problem
Our BE currently is in JS, which is a dynamically and weakly typed language. this could cause potential problems for us down the line, especially when the codebase grows and becomes more complicated 

## Solution
This PR adds a **conservative** `tsconfig`, where i attempt to stick as strictly as possible to the original `jsconfig`. Hence, i didn't opt for stuff like `module: ESNext`. If there is a need to, we can in the future but to avoid potential build issues, this was not done now.

## Tests
1. set `noEmit` in the `tsconfig` to `true`
2. set `outDir` in the `tsconfig` to `/build`
3. run `npx tsc` - this should work

## devDeps
